api/DHSC.FingertipsNext.Modules.Area/Controllers/V1/AreaController.cs

```diff
@@ -89,16 +89,16 @@ public async Task<IActionResult> GetAreaDetailsAsync(
     /// <summary>
     /// Get the areas that have a given area type
     /// </summary>
-    /// <param name="area_type"></param>
+    /// <param name="area_type_key"></param>
     /// <returns></returns>
     [HttpGet]
     [ProducesResponseType(typeof(List<Schemas.Area>), StatusCodes.Status200OK)]
     [ProducesResponseType(StatusCodes.Status404NotFound)]
     [Route("areatypes/{area_type}/areas")]
     [SuppressMessage("ReSharper", "InconsistentNaming")]
-    public async Task<IActionResult> GetAreaDetailsForAreaTypeAsync([FromRoute] string area_type)
+    public async Task<IActionResult> GetAreaDetailsForAreaTypeAsync([FromRoute] string area_type_key)
```

Comment:
Can we change this to camelCase please as per Microsoft Coding Standards.  Appreciate it was already in snake_case though.

Reply:
Resolving following a conversation with Howard as this has wider implications.